website/project/views/node.py

```diff
@@ -660,6 +660,7 @@ def _view_project(node, auth, primary=False):
                 status.push_status_message(message, kind='info', dismissible=False, trust=True)
     data = {
         'node': {
+            'disapproval_link': node.registration_approval.stashed_urls.get(user._id, {}).get('reject', '') if (node.is_pending_registration and node.has_permission(user, ADMIN)) else '',
```

Comment:
This is a little more than I usually like to put on a single line. How about breaking up some of the data gathering into the method above and keeping this line a little simpler? Not a big deal.